index.js

```diff
@@ -107,15 +107,24 @@ class UpdateNotifier {
 			};
 		});
 	}
+	isYarnGlobal() {
+		const isWindows = process.platform === 'win32' || process.env.OSTYPE === 'cygwin' || process.env.OSTYPE === 'msys';
+		const yarnPath = isWindows ? path.join('Yarn', 'config', 'global') : path.join('.config', 'yarn', 'global');
+		if (process.cwd().includes(yarnPath)) {
+			return true;
+		}
+		return false;
+	}
 	notify(opts) {
 		if (!process.stdout.isTTY || isNpm() || !this.update) {
 			return this;
 		}
 
 		opts = Object.assign({isGlobal: isInstalledGlobally()}, opts);
```

Comment:
`isInstalledGlobally` already supports Yarn, but it doesn't return which one it is. For that, you could use https://github.com/sindresorhus/global-dirs#packages

Reply:
I created a package [is-yarn-global](https://github.com/LitoMore/is-yarn-global), it without any `fs` calls and dependencies. What about this?